be/src/runtime/memory/chunk_allocator.cpp

```diff
@@ -204,7 +205,7 @@ Status ChunkAllocator::allocate(size_t size, Chunk* chunk) {
 void ChunkAllocator::free(const Chunk& chunk) {
     DCHECK(chunk.core_id != -1);
     CHECK((chunk.size & (chunk.size - 1)) == 0);
-    if (config::disable_mem_pools) {
+    if (config::disable_chunk_allocator) {
```

Comment:
Is it better to move the condition into `MemPool` and rename `disable_chunk_allocator_in_mem_pool`

Reply:
Not use this. I will remove disable_mem_pools in the future after non-vectorized engine is removed.  It is very confused with MemPool.

Reply:
Try removing `disable_chunk_allocator_in_vec` and replace with `disable_chunk_allocator`. (more detailed config comments)